OSDOCS-19307 created zstream RNs for 4-16-60 (#110879)
openshift/openshift-docs: modules/zstream-4-16-60.adoc

// Module included in the following assemblies:
//
// * release_notes/ocp-4-16-release-notes.adoc

:_mod-docs-content-type: REFERENCE
[id="zstream-4-16-60_{context}"]
= RHSA-2026:10104 - {product-title} {product-version}.60 fixed issues and security updates

Issued: 20 April 2026

[role="_abstract"]
{product-title} release {product-version}.60 is now available. The list of fixed issues that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2026:10104[RHSA-2026:10104] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2026:10096[RHSA-2026:10096] advisory.

Space precluded documenting all of the container images for this release in the advisory.

You can view the container images in this release by running the following command:

[source,terminal]
----
$ oc adm release info 4.16.60 --pullspecs
----

[id="zstream-4-16-60-fixed-issues_{context}"]
== Fixed issues

> * Before this update, the cluster autoscaler would process paused node groups as if they were active, which could lead to the wrong nodes being deleted. With this release, the cluster autoscaler identifies paused node groups and does not act on them. (link:https://redhat.atlassian.net/browse/OCPBUGS-78702[OCPBUGS-78702])

Reviewer: ```suggestion
* Before this update, the cluster autoscaler processed paused node groups as if they were active, which led to the wrong nodes being deleted. With this release, the cluster autoscaler identifies paused node groups and does not act on them. (link:https://redhat.atlassian.net/browse/OCPBUGS-78702[OCPBUGS-78702])
```
maybe a renoa thing, but we don't use past perfect, really; just past and present

> * Before this update, cluster namespaces with the "nodes" suffix caused the Performance Profile Creator (PPC) to fail. The PPC mistook `namespace-node` directories with the `must-gather` tool `nodes` directories in the `must-gather` data processed by the PPC. With this release, the PPC now correctly excludes the `namespaces` directory when processing the `must-gather` data to create a suggested `PerformanceProfile`. (link:https://redhat.atlassian.net/browse/OCPBUGS-78766][OCPBUGS-78766])

Reviewer 2: ```suggestion
* Before this update, cluster namespaces with the "nodes" suffix caused the Performance Profile Creator (PPC) to fail. The PPC mistook `namespace-node` directories with the `must-gather` tool `nodes` directories in the `must-gather` data processed by the PPC. With this release, the PPC now correctly excludes the `namespaces` directory when processing the `must-gather` data to create a suggested `PerformanceProfile`. (link:https://redhat.atlassian.net/browse/OCPBUGS-78766[OCPBUGS-78766])
```
@wgabor0427 one more little bracket still needs to be removed

Author: OOOPS! Sorry removed

> * Before this update, cluster namespaces with the "nodes" suffix caused the Performance Profile Creator (PPC) to fail. The PPC mistook `namespace-node` directories with the `must-gather` tool `nodes` directories in the `must-gather` data processed by the PPC. With this release, the PPC now correctly excludes the `namespaces` directory when processing the `must-gather` data to create a suggested `PerformanceProfile`.   (link:[https://redhat.atlassian.net/browse/OCPBUGS-73797|https://redhat.atlassian.net/browse/OCPBUGS-78766][OCPBUGS-78766])

Reviewer 2: ```suggestion
* Before this update, cluster namespaces with the "nodes" suffix caused the Performance Profile Creator (PPC) to fail. The PPC mistook `namespace-node` directories with the `must-gather` tool `nodes` directories in the `must-gather` data processed by the PPC. With this release, the PPC now correctly excludes the `namespaces` directory when processing the `must-gather` data to create a suggested `PerformanceProfile`.  (link:https://redhat.atlassian.net/browse/OCPBUGS-78766[OCPBUGS-78766])
```

Author: Done

> * Before this update, cluster namespaces with "nodes" suffix in their name would cause the Performance Profile Creator (PPC) to fail when mistaking incorrect `namespace` directories with the must-gather `nodes` directories in the must-gather processed by the PPC. With this release, the PPC now correctly excludes the `namespaces` directory when processing the must-gather data to create a suggested `PerformanceProfile`.   (link:[https://redhat.atlassian.net/browse/OCPBUGS-73797|https://redhat.atlassian.net/browse/OCPBUGS-78766][OCPBUGS-78766])

Reviewer: ```suggestion
* Before this update, cluster namespaces with the `-nodes` suffix caused the Performance Profile Creator (PPC) to fail. The PPC mistook `namespace-node` directories with the `must-gather` tool `nodes` directories in the `must-gather` data processed by the PPC. With this release, the PPC now correctly excludes the `namespaces` directory when processing the `must-gather` data to create a suggested `PerformanceProfile`.  (link:[https://redhat.atlassian.net/browse/OCPBUGS-73797|https://redhat.atlassian.net/browse/OCPBUGS-78766][OCPBUGS-78766])
```
see https://docs.redhat.com/en/documentation/openshift_container_platform/4.21/html/support/gathering-cluster-data

Author: Done
